SimPEG/DataMisfit.py

```diff
@@ -47,12 +47,12 @@ def evalDeriv(self, m, u=None):
 
 
     @Utils.timeIt
-    def eval2Deriv(self, m, v, u=None):
-        """eval2Deriv(m, v, u=None)
+    def eval2Deriv(self, m, v, f=None):
+        """eval2Deriv(m, v, f=None)
 
             :param numpy.array m: geophysical model
             :param numpy.array v: vector to multiply
-            :param numpy.array u: fields
+            :param Fields f: fields
```

Comment:
This does link up properly (SimPEG.Fields doesn't) 
<img width="1261" alt="data misfit simpeg 0 1 10 documentation 2016-03-31 23-56-35 png 2016-03-31 23-58-21" src="https://cloud.githubusercontent.com/assets/6361812/14199836/8c2f81f6-f79c-11e5-9701-00ef66ce0709.png">